home-work-app/src/app/pages/movie/components/Description/Description.tsx

```diff
@@ -1,7 +1,59 @@
-import { StyledDescription } from './StyledDescription.styled';
+import { MovieDetails } from './MovieDetails';
+import { useParams } from 'react-router-dom';
+import { IApiResponse } from '../../../../interfaces/IApiResponse';
+import { movieResponse } from '../../../../../../tempData/getMoviesResponse';
+import { find } from 'lodash';
+import { MovieDetailsPoster } from './MovieDetailsPoster.styled';
+import { MovieDetailsContent } from './MovieDetailsContent.styled';
+import { MovieDetailsHeader } from './MovieDetailsHeader';
+import { VoteAverage } from './VoteAverage.styled';
+import { MovieDetailsTitle } from './MovieDetailsTitle.styled';
+import { MovieDetailsTagline } from './MovieDetailsTagline.styled';
+import { MovieDetailsTimeData } from './MovieDetailsTimeData.styled';
+import { MovieDetailsOverview } from './MovieDetailsOverview.styled';
 
-export const Description: React.FC = () => (
-    <StyledDescription>
-        Movie page
-    </StyledDescription>
-);
+const movieCollection: IApiResponse.IMovie[] = movieResponse.data;
+
+export const Description: React.FC = () => {
+    const {id} = useParams();
+    const movie: IApiResponse.IMovie = find(movieCollection, {id: Number(id)});
+    const {
+        title,
+        overview,
+        tagline,
+        poster_path,
+        runtime,
+        release_date,
+        vote_average
+    } = movie;
+
+    return (
+        <MovieDetails>
+            <MovieDetailsPoster src={poster_path} alt={title}/>
+            <MovieDetailsContent>
+                <MovieDetailsHeader>
+                    <MovieDetailsTitle>
```

Comment:
From my point of view, passing data with prop is much more cleaner, understandable and "reactive" than composition.
```
<MovieDetails>
     <MovieDetailsPoster src={poster_path} alt={title}/>
        <MovieDetailsContent>
             <MovieDetailsHeader>
                <MovieDetailsTitle title={title} />
                <VoteAverage voteAverage ={vote_average} />
            </MovieDetailsHeader>
            <MovieDetailsTagline tagline={tagline} />
            <MovieDetailsTimeData year={new Date(release_date).getFullYear()} />
            <MovieDetailsOverview overview={overview} />
   </MovieDetailsContent>
</MovieDetails>
```